provision/testdata/base_case.kube.yaml

```diff
@@ -248,6 +248,19 @@ data:
     }
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: snat-operator-config
+  namespace: aci-containers-system
+  labels:
+    aci-containers-config-version: "dummy"
+    network-plugin: aci-containers
+data:
+    "start": "5000"
+    "end": "65000"
+    "portsPerNode": "3000"
```

Comment:
Why are these strings, should they not be integers?

Reply:
K8s doesn't allow you to add integer values in ConfigMap in this format. Either we use a JSON file like we have used in the configMap, which we moved away from because "getting" it directly from k8s client was giving a long string. 
ref: https://medium.com/@ismailyenigul/kubernetes-configmap-and-cannot-convert-int64-to-string-error-ebf48fa1cd28